cleverhans/utils_tf.py

```diff
@@ -274,6 +274,9 @@ def tf_model_load(sess, file_path=None):
     with sess.as_default():
         saver = tf.train.Saver()
         if file_path is None:
+            warnings.warn("FLAGS argument is deprecated. "
+                          "Please provide file_path argument. "
+                          "FLAGS argument will be removed after 2018-04-23.")
```

Comment:
Can you change the `FLAGS argument` part to `FLAGS.train_dir` and `FLAGS.filename`